kotlin/workflow-runtime/src/main/java/com/squareup/workflow/internal/Workers.kt

```diff
@@ -131,8 +133,9 @@ private fun <T> Flow<T>.transformToValueOrDone(): Flow<ValueOrDone<T>> = flow {
 
 private fun CoroutineScope.createWorkerScope(
   worker: Worker<*>,
-  key: String
-): CoroutineScope = this + CoroutineName(worker.debugName(key)) + Unconfined
+  key: String,
+  workerContext: CoroutineContext
+): CoroutineScope = this + CoroutineName(worker.debugName(key)) + Unconfined + workerContext
```

Comment:
Ahh, I see what you meant when you were describing `CouroutineScope` at your talk.